api/tests/identity-access-management/acceptance/application/anonymization.admin.route.test.js

```diff
@@ -0,0 +1,71 @@
+import iconv from 'iconv-lite';
+
+import {
+  createServer,
+  databaseBuilder,
+  expect,
+  generateValidRequestAuthorizationHeader,
+  insertUserWithRoleSuperAdmin,
+} from '../../../test-helper.js';
+
+describe('Acceptance | Identity Access Management | Application | Route | Admin | Anonymization', function () {
+  let server;
+
+  beforeEach(async function () {
+    server = await createServer();
+  });
+
+  describe('POST /api/admin/anonymize/gar', function () {
+    context('when a CSV file is loaded', function () {
+      it('responds with a 200 and serialized payload', async function () {
+        // given
+
+        const user = await insertUserWithRoleSuperAdmin();
+
+        const userId1 = databaseBuilder.factory.buildUser().id;
+        const userId2 = databaseBuilder.factory.buildUser().id;
+        const userId3 = databaseBuilder.factory.buildUser().id;
+
+        databaseBuilder.factory.buildAuthenticationMethod.withGarAsIdentityProvider({
+          userId: userId1,
+          externalIdentifier: 'externalId1',
+        });
+        databaseBuilder.factory.buildAuthenticationMethod.withGarAsIdentityProvider({
+          userId: userId2,
+          externalIdentifier: 'externalId2',
+        });
+        databaseBuilder.factory.buildAuthenticationMethod.withGarAsIdentityProvider({
+          userId: userId3,
+          externalIdentifier: 'externalId3',
+        });
+        await databaseBuilder.commit();
+
+        const input = `User ID
+      ${userId1}
+      ${userId2}
+      ${userId3}
+      `;
+
+        const options = {
+          method: 'POST',
+          headers: { authorization: generateValidRequestAuthorizationHeader(user.id) },
+          url: '/api/admin/anonymize/gar',
+          payload: iconv.encode(input, 'UTF-8'),
```

Comment:
Ici on peut se passer de la conversion en UTF-8 puisque la 1ère ligne contient uniquement du texte en ASCII et que les lignes suivantes contiennent uniquement des digits. En effet la conversion en UTF-8 de ce contenu sera la même que le contenu d'origine.

```suggestion
          payload: input,
```

Reply:
Ce point n'a pas été modifié, mais on pourra ajouter un commit dans une autre PR.